babel-watch.js

```diff
@@ -222,43 +222,42 @@ function generateTempFilename() {
   ].join(''));
 }
 
-function handleFileLoad(filename, callback) {
+async function handleFileLoad(filename) {
   const cached = cache[filename];
   if (cached) {
     const stats = fs.statSync(filename);
     if (stats.mtime.getTime() === cached.mtime) {
-      callback(cache[filename].code, cache[filename].map);
-      return;
+      return [cache[filename].code, cache[filename].map];
     }
   }
   if (!shouldIgnore(filename)) {
```

Comment:
Since we're refactoring, might as well collapse the condition

Reply:
Good call, much more clear